File: web/src/api/pas/evaluationKpiUser.js
import service from '@/utils/request'

 export const findEvaluationKpiUser = (data) => {
     return service({
         url: "/EvaluationKpiUser/findEvaluationKpiUser",
         method: 'post',
         data
     })
 }

 export const createEKU = (data) => {
    return service({
        url: "/EvaluationKpiUser/createEKU",
        method: 'post',
        data
    })
}
export const getEKUByEKID = (data) => {
    return service({
        url: "/EvaluationKpiUser/getEKUByEKID",
        method: 'post',
        data
    })
}
export const updateEKU = (data) => {
    return service({
        url: "/EvaluationKpiUser/updateEKU",
        method: 'PUT',
        data
    })
}

File: server/router/pas/evaluation_kpi_user.go
package pas

import (
	"gin-vue-admin/api/v1/pas"
	"gin-vue-admin/middleware"
	"github.com/gin-gonic/gin"
)

func InitEvaluationKpiUserRouter(Router *gin.RouterGroup) {
	EvaluationKpiUserRouter := Router.Group("EvaluationKpiUser").Use(middleware.JWTAuth()).Use(middleware.CasbinHandler()).Use(middleware.OperationRecord())
	{
		EvaluationKpiUserRouter.POST("createEKU", pas.CreateEKU)   // 新建Kpi
		EvaluationKpiUserRouter.POST("getEKUByEKID", pas.GetEKUByEKID)   // 新建Kpi
		EvaluationKpiUserRouter.PUT("updateEKU", pas.UpdateEKU)   // 新建Kpi
		
		
	}
}
